sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Bugs Fixed
 
+- Throw std::invalid_argument if the `TimeFractionFormat` enum passed in to DateTime `ToString()` is invalid.
```

Comment:
Yeah, I considered that, but given no valid customer facing scenario is impacted by this, I wanted to reduce the noise-to-signal ratio and landed on a bug fix.

This catches a potential bug, even though it might be technically behaviorally breaking. Alerting all users of such a change as breaking, even though almost no one will have any action to take, didn't feel like the right trade-off.